uvicorn/loops/asyncio.py

```diff
@@ -1,7 +1,12 @@
 import asyncio
+import logging
 import sys
+from typing import Any
 
+logger = logging.getLogger("uvicorn.error")
 
-def asyncio_setup() -> None:  # pragma: no cover
-    if sys.version_info >= (3, 8) and sys.platform == "win32":
+
+def asyncio_setup(reload: bool = False, **_: Any) -> None:  # pragma: no cover
```

Comment:
is there really a need to declare unused kwargs here ?

Reply:
@euri10 No. I was unnecessarily avoiding an unused argument linter error.

Updated to explicit reload keyword argument in the next revision.